changes/9570.pipeline.rst

```diff
@@ -0,0 +1 @@
+Lazy-load metadata where possible in calwebb_image3; result is a few percent runtime improvement if in_memory=False
```

Comment:
```suggestion
Use ``read_metadata`` where possible in calwebb_image3; result is a few percent runtime improvement if in_memory=False
```
"Lazy" makes me think it's only loaded when used. Let's use `read_metadata` here (in part to advertise the method). Maybe some sphinx-fu could make that link work in the docs but I will not risk writing sphinx compatible code in a comment suggestion.